eventually/task/models.py

```diff
@@ -67,16 +100,20 @@ def to_dict(self):
             'status': 1,
             'created_at': 1509540116,
             'updated_at': 1509540116,
+            'event' : 13,
+            'users' : [21, 33]
         }
         """
 
         return {
-            'id':self.id,
+            'id': self.id,
             'title': self.title,
             'description': self.description,
             'status': self.status,
             'created_at': int(self.created_at.timestamp()),
             'updated_at': int(self.updated_at.timestamp()),
+            'event': self.event.id,
+            'users': [user.id for user in self.users.all()]
         }

```

Comment:
it is better to write fields just under "{" . And put "}" behind "]" like this "'users': [user.id for user in self.users.all()]}"

Reply:
JS